Vagrant installation not working

I'm following the instructions on https://flynn.io/docs/installation/vagrant and using commit 168108422f6bfc12132d56a9ca9db33b64d4d218 but I get this error when doing `make init`:

```
linux-headers-3.19.0-39-generic is already the newest version.
Suggested packages:
  debhelper
The following packages will be DOWNGRADED:
  dkms
0 upgraded, 0 newly installed, 1 downgraded, 0 to remove and 47 not upgraded.
Need to get 72.4 kB of archives.
After this operation, 1,024 B disk space will be freed.
E: There are problems and -y was used without --force-yes
==> default: Checking for guest additions in VM...
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!

apt-get install -y linux-headers-`uname -r` dkms

Stdout from the command:

Reading package lists...
Building dependency tree...
Reading state information...
linux-headers-3.19.0-39-generic is already the newest version.
Suggested packages:
  debhelper
The following packages will be DOWNGRADED:
  dkms
0 upgraded, 0 newly installed, 1 downgraded, 0 to remove and 47 not upgraded.
Need to get 72.4 kB of archives.
After this operation, 1,024 B disk space will be freed.


Stderr from the command:

stdin: is not a tty
E: There are problems and -y was used without --force-yes

make[1]: *** [up] Error 1
make: *** [init] Error 2
```

--- a/Vagrantfile
+++ b/Vagrantfile
@@ -29,6 +29,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box_url = "https://dl.flynn.io/vagrant/flynn-base.json"
   config.vm.box_version = "> 0"
 
+  if Vagrant.has_plugin?("vagrant-vbguest")
+    # vagrant-vbguest can cause the VM to not start: https://github.com/flynn/flynn/issues/2874
+    config.vbguest.auto_update = false
+  end
+
   # VAGRANT_MEMORY          - instance memory, in MB
   # VAGRANT_CPUS            - instance virtual CPUs
   config.vm.provider "virtualbox" do |v, override|

--- a/demo/Vagrantfile
+++ b/demo/Vagrantfile
@@ -27,6 +27,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
 
   config.vm.synced_folder ".", "/vagrant", disabled: true
 
+  if Vagrant.has_plugin?("vagrant-vbguest")
+    # vagrant-vbguest can cause the VM to not start: https://github.com/flynn/flynn/issues/2874
+    config.vbguest.auto_update = false
+  end
+
   config.vm.provider "virtualbox" do |v|
     v.memory = ENV["VAGRANT_MEMORY"] || 2048
     v.cpus = ENV["VAGRANT_CPUS"] || 2

